curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bcc8ccc976fd791610f43.md

```diff
@@ -10,6 +10,38 @@ dashedName: what-is-the-css-transform-property
+
+The transform property works by applying a mathematical transformation to an elements coordinate system. This means you can manipulate an element's shape and position while keeping it's original place and document flow intact.
+
+Let's explore some common transform functions. Here's an example of a box element, we have set the body to have a solid black border so that you can see the box element nested inside the body element. 
+
+The translate function moves an element from its current position. Here's an updated example using the translate function. This CSS rule will move the element with the class box 50 pixels to the right and 100 pixels down from its original position. Here's what the modified result looks like on the screen.
+
+The rotate function rotates an element around a fixed point and this is an example of using the rotate function for the box element from earlier. This will rotate the element forty five degrees clockwise and here's what it looks like.
+
+The scale function allows you to change the size of an element. Here's an example, this will make the element one and a half times wider and twice as tall as it's original size, and this is what it looks like.
+
+You can combine multiple transformations in a single declaration. This will move the element 50 pixels to the right and down, rotate it 45 degrees, and scale it to be one and a half times it's original size and here's the result.
+
+While the transform property is powerful for creating visually appealing designs, it's important to consider accessibility when using it. Here are some important accessibility concerns to keep in mind: 
+
+Screen readers may not accurately convey transformed content. For example, if you use transform to rearrange the visual order of elements, screen readers will still read the content in the original DOM order, this can lead to confusion for users relying on screen readers.
+
+When using scale to resize text be cautious not to make it too small or too large. Extremely small text can be difficult to read while overly large text might overflow its container and become unreadable. It's generally better to use proper font styling techniques for text resizing.
+
+If you are using transform for animations effects, be mindful to users who are sensitive to motion. Excessive or rapid animations can cause discomfort or even trigger seizures for some people. Consider providing a way for users to reduce or turn off animations. When using 3D transforms, remember that not all users perceive depth in the same way. Ensure that critical information conveyed through 3D effects is also available in a 2D format or through text.
+
+If you are using transform to hide or reveal content, make sure the content is still accessible to screen readers and keyboard navigation. Hidden content should be truly hidden such as by using display none or visibility hidden, rather than just being visually moved offscreen. 
```

Comment:
```suggestion
If you are using `transform` to hide or reveal content, make sure the content is still accessible to screen readers and keyboard navigation. Hidden content should be truly hidden such as by using `display: none;` or `visibility: hidden;`, rather than just being visually moved offscreen. 
```